Rubberduck.Inspections/QuickFixes/RemoveEmptyConditionBlockQuickFix.cs

```diff
@@ -92,6 +92,16 @@ private void UpdateContext(VBAParser.ElseIfBlockContext context, IModuleRewriter
             rewriter.Remove(context);
         }
 
+        private void UpdateContext(VBAParser.ElseBlockContext context, IModuleRewriter rewriter)
+        {
+            VBAParser.BlockContext elseBlock = context.block();
```

Comment:
Why the explicit type here?

Reply:
I think it was carried over from when they were in separate files.